homeassistant/components/tankerkoenig/__init__.py

```diff
@@ -195,7 +199,7 @@ def setup(self) -> bool:
                     station_id,
                     station_data["message"],
                 )
-                return False
+                continue
```

Comment:
The original reason to return False here was (in the yaml version) to make the user aware of wrong/unexisting station IDs.
It was originally requested by a reviewer at that point, to fail instead of just skip the station in this case.

With the UI config flow, this cannot happen (except on if some of the yaml-imported stations does not exist anymore), so this change makes sense.
(Just wanted to give my 2cts as the original developer ;-) )